### PR TITLE
Cypress/fe-2012 regression update

### PR DESCRIPTION
**Pull request by marcinstachowicz**
### Regression automation tests was updated and adjusted according to development

### CODE REVIEW

File: cypress/features/regression/pages.feature [hunk 1]
```diff
@@ -1,10 +1,11 @@
+# Tests are ignore due to development waiting phase. Scenarios need to be refactor after merge FE-1587
```
Comment (DlgSHi) [inline cypress/features/regression/pages.feature]: The branch `FE-1587` is already merged

File: cypress/features/regression/profileClassic.feature [hunk 2]
```diff
@@ -20,7 +21,7 @@ Feature: Profile component
       And avatar is taken from "<avatar>"
     Examples:
       | email                | avatar                                                                        |
-      | andrew.tait@sage.com | https://www.gravatar.com/avatar/ec55ecf2e1c7e7e56a904b50245c24a4?s=40&d=blank |
+      | andrew.tait@sage.com | https://www.gravatar.com/avatar/ec55ecf2e1c7e7e56a904b50245c24a4?s=40&d=404 |
```
Comment (DlgSHi) [inline cypress/features/regression/profileClassic.feature]: Why did You change to 404 at the end of this line?

File: cypress/features/regression/showEditPodClassic.feature [hunk 3]
```diff
@@ -134,7 +134,7 @@ Feature: Show Edit Pod component
   Scenario Outline: Set Show Edit Pod buttonAlign to <buttonAlign>
     When I select buttonAlign to "<buttonAlign>"
       And I edit Show Edit Pod component
-    Then Show Edit Pod buttonAlign value is set to "<buttonAlign>"
+    Then Show Edit Pod buttons are aligned <buttonAlign>
```
Comment (DlgSHi) [inline cypress/features/regression/showEditPodClassic.feature]: Grammatically correct should be "Show Edit Pod buttons are aligned to <buttonAlign>"

File: cypress/locators/show-edit-pod/locators.js [hunk 4]
```diff
@@ -4,7 +4,8 @@ export const SHOW_EDIT_POD_TITLE = 'h4[data-element=title]';
 export const SHOW_EDIT_POD_COLLAPSIBLE_CONTENT = 'div[class="carbon-pod__collapsible-content"] > div > div > div';
 export const SHOW_EDIT_SAVE_BUTTON = 'button[data-element="save"]';
 export const SHOW_EDIT_CANCEL_BUTTON = 'button[data-element="cancel"]';
-export const SHOW_EDIT_DELETE_BUTTON = 'error > span > span';
+export const SHOW_EDIT_DELETE_BUTTON = '.carbon-link__content';
```
Comment (DlgSHi) [inline cypress/locators/show-edit-pod/locators.js]: does the edit delete button have another attribute to locate it?

File: cypress/support/step-definitions/icon-steps.js [hunk 5]
```diff
@@ -1,7 +1,16 @@
...
+  icon().should('have.css', 'height', height);
+});
 
-Then('bg{word} is set to {string}', (word, bg) => {
-  icon().should('have.class', `${ICON_PREFIX}${bg}`);
+Then('radius is set to {string}', (radius) => {
+  icon().should('have.css', 'border-bottom-left-radius', radius);
+  icon().should('have.css', 'border-bottom-right-radius', radius);
+  icon().should('have.css', 'border-top-left-radius', radius);
+  icon().should('have.css', 'border-top-left-radius', radius);
```
Comment (DlgSHi) [inline cypress/support/step-definitions/icon-steps.js]: we need to use twice `left` radius?

File: cypress/support/step-definitions/show-edit-pod-steps.js [hunk 6]
```diff
@@ -96,9 +95,12 @@ Then('Show Edit Pod as value is set to {string}', (property) => {
     .should('have.class', `${SHOW_EDIT_POD_CONTENT}${property}`);
 });
 
-Then('Show Edit Pod buttonAlign value is set to {string}', (property) => {
-  showEditPodSecondaryWrapper()
-    .should('have.class', `${SHOW_EDIT_POD_WRAPPER}${property}`);
+Then('Show Edit Pod buttons are aligned left', () => {
+  showEditPodFooter().children().first().should('have.text', 'Save');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/show-edit-pod-steps.js]: if we are using this form like `showEditPodFooter().children()` only in those 2 steps, probably we should move the `children()` part to `index.js` file and after that use only `showEditPodFooter()`, what do you think about that?

Comment (marcinstachowicz) [reply cypress/support/step-definitions/show-edit-pod-steps.js]: fixed

File: cypress/features/regression/profileClassic.feature (hunk 2 above)
Comment (marcinstachowicz) [reply cypress/features/regression/profileClassic.feature]: because it works that way 👾

File: cypress/features/regression/showEditPodClassic.feature (hunk 3 above)
Comment (marcinstachowicz) [reply cypress/features/regression/showEditPodClassic.feature]: fixed

File: cypress/support/step-definitions/icon-steps.js (hunk 5 above)
Comment (marcinstachowicz) [reply cypress/support/step-definitions/icon-steps.js]: fixed
